app/packs/stylesheets/bootstrapOverrides.scss

```diff
@@ -8,3 +8,8 @@
 .table {
   border: 1px solid $table-border-color;
 }
+
+// Better contrast for accessability
+.form-text {
```

Comment:
Instead of overriding, set $form-text-color (see https://github.com/twbs/bootstrap/blob/f61a0218b36d915db80dc23635a9078e98e2e3e0/scss/forms/_form-text.scss#L10)

Reply:
I went looking for that. Where do you find those variables documented?